src/app/map-chart/map-chart.component.ts

```diff
@@ -0,0 +1,297 @@
+        ['co', 80],
+        ['tl', 81],
+        ['tz', 82],
+        ['ar', 83],
+        ['sa', 84],
+        ['pk', 85],
+        ['ye', 86],
+        ['ae', 87],
+        ['ke', 88],
+        ['pe', 89],
+        ['do', 90],
+        ['ht', 91],
+        ['pg', 92],
+        ['ao', 93],
+        ['kh', 94],
+        ['vn', 95],
+        ['mz', 96],
+        ['cr', 97],
+        ['bj', 98],
+        ['ng', 99],
+        ['ir', 100],
+        ['sv', 101],
+        ['sl', 102],
+        ['gw', 103],
+        ['hr', 104],
+        ['bz', 105],
+        ['za', 106],
+        ['cf', 107],
+        ['sd', 108],
+        ['cd', 109],
+        ['kw', 110],
+        ['de', 111],
+        ['be', 112],
+        ['ie', 113],
+        ['kp', 114],
+        ['kr', 115],
+        ['gy', 116],
+        ['hn', 117],
+        ['mm', 118],
+        ['ga', 119],
+        ['gq', 120],
+        ['ni', 121],
+        ['lv', 122],
+        ['ug', 123],
+        ['mw', 124],
+        ['am', 125],
+        ['sx', 126],
+        ['tm', 127],
+        ['zm', 128],
+        ['nc', 129],
+        ['mr', 130],
+        ['dz', 131],
+        ['lt', 132],
+        ['et', 133],
+        ['er', 134],
+        ['gh', 135],
+        ['si', 136],
+        ['gt', 137],
+        ['ba', 138],
+        ['jo', 139],
+        ['sy', 140],
+        ['mc', 141],
+        ['al', 142],
+        ['uy', 143],
+        ['cnm', 144],
+        ['mn', 145],
+        ['rw', 146],
+        ['so', 147],
+        ['bo', 148],
+        ['cm', 149],
+        ['cg', 150],
+        ['eh', 151],
+        ['rs', 152],
+        ['me', 153],
+        ['tg', 154],
+        ['la', 155],
+        ['af', 156],
+        ['ua', 157],
+        ['sk', 158],
+        ['jk', 159],
+        ['bg', 160],
+        ['qa', 161],
+        ['li', 162],
+        ['at', 163],
+        ['sz', 164],
+        ['hu', 165],
+        ['ro', 166],
+        ['ne', 167],
+        ['lu', 168],
+        ['ad', 169],
+        ['ci', 170],
+        ['lr', 171],
+        ['bn', 172],
+        ['iq', 173],
+        ['ge', 174],
+        ['gm', 175],
+        ['ch', 176],
+        ['td', 177],
+        ['kv', 178],
+        ['lb', 179],
+        ['dj', 180],
+        ['bi', 181],
+        ['sr', 182],
+        ['il', 183],
+        ['ml', 184],
+        ['sn', 185],
+        ['gn', 186],
+        ['zw', 187],
+        ['pl', 188],
+        ['mk', 189],
+        ['py', 190],
+        ['by', 191],
+        ['cz', 192],
+        ['bf', 193],
+        ['na', 194],
+        ['ly', 195],
+        ['tn', 196],
+        ['bt', 197],
+        ['md', 198],
+        ['ss', 199],
+        ['bw', 200],
+        ['bs', 201],
+        ['nz', 202],
+        ['cu', 203],
+        ['ec', 204],
+        ['au', 205],
+        ['ve', 206],
+        ['sb', 207],
+        ['mg', 208],
+        ['is', 209],
+        ['eg', 210],
+        ['kg', 211],
+        ['np', 212]
+      ]
+    } as Highcharts.SeriesMapOptions,
+    {
+      // Specify points using lat/lon
```

Comment:
While this is not caused by this PR it would be appreciated if you could improve this series visibility in the chart. For example, disable other series dataLabels, change color for the points to make them more visible, set chart title or subtitle with info about this lat/long demonstration, make tooltip format nicer, etc. - at some of the changes like this should be enough.